src/plugins/dashboard/public/dashboard_api/types.ts

```diff
@@ -153,7 +153,7 @@ export type DashboardApi = CanExpandPanels &
     focusedPanelId$: PublishingSubject<string | undefined>;
     forceRefresh: () => void;
     getSettings: () => DashboardStateFromSettingsFlyout;
-    getDashboardPanelFromId: (id: string) => Promise<DashboardPanelState>;
+    getDashboardPanelFromId: (id: string) => DashboardPanelState;
```

Comment:
src/plugins/dashboard/public/dashboard_actions/copy_to_dashboard_modal.tsx still awaits `getDashboardPanelFromId`.

Reply:
removed in 8b0809cccd7